urbanairship-chat-react-native/ios/AirshipChatModule.m

```diff
@@ -24,21 +24,21 @@ - (instancetype)init{
 RCT_EXPORT_METHOD(openChat) {
     [[NSOperationQueue mainQueue] addOperationWithBlock:^ {
         if (@available(iOS 13.0, *)) {
-            [[UAirshipChat shared] openChat];
+            [[UAChat shared] openChat];
```

Comment:
we need to wire up the openConnection method

Reply:
Do we talk about `[[UAChat shared].conversation connect]` ?

Reply:
I think @Apekka left a react method in by mistake, but its new to this version. People need to connect during displaying there custom UI